inlineplz/linters/gherkinlint.py

```diff
@@ -4,10 +4,37 @@
 import traceback
 
 import dirtyjson as json
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "import dirtyjson as json" (wrong-import-order)`